### ISSUE TITLE
arium.events

### ISSUE THREAD

**Issue by AidanNelson** [hoshsadiq/adblock-nocoin-list#411]
Hello there,

My legitimate websocket connections are being blocked by this list.  Could you let me know how to whitelist this site?

@hoshsadiq fixed [another issue](https://github.com/hoshsadiq/adblock-nocoin-list/issues/408) that I opened for a similar site a few months ago.  Maybe you could fix this one too?  

The site is https://arium.events/ and the connections that are being blocked are between several pages and the backend servers.  The backend servers are [staging-server.vlts.pw](staging-server.vlts.pw) and [server.vlts.pw](server.vlts.pw).

**Comment by smed79** [comment]
Hi, 

Exception added for `arium.events` and `yorbbackend.xyz` (https://github.com/hoshsadiq/adblock-nocoin-list/issues/408#issuecomment-697971770).
